frontend/src/api/calls/skillTypeCalls.ts

```diff
@@ -8,8 +8,16 @@ import {
     ISkillTypePage,
     skillTypeCollectionName,
 } from "../entities/SkillTypeEntity";
-import { AxiosConf, getAllEntitiesFromLinksUrl } from "./baseCalls";
+import { AxiosConf, getAllEntitiesFromLinksUrl, getAllEntitiesFromPage } from "./baseCalls";
 import { IUser, userCollectionName } from "../entities/UserEntity";
+import { IProject, projectCollectionName } from "../entities/ProjectEntity";
+
+/**
+ * Fetches all SkillTypes on a given SkillTypePageUrl
+ */
+export function getAllSkillTypesFormPage(url: string): Promise<ISkillType[]> {
```

Comment:
```suggestion
export function getAllSkillTypesFromPage(url: string): Promise<ISkillType[]> {
```
typo in function name, you should change this in webstorm or something as just commiting this suggestion will probably break stuff.